:memo: Add instructions to fix cygwin symlink bug on Windows (#30)
uclcshub/uclcshub.github.io: docs/set-up-environment/windows/c.mdx

![Cygwin!](/img//windows-cygwin.png)

<details>
<summary> Let's test your installation!</summary>

Type `gcc` in your shell. You should receive an `error: no input files` message.

If _not_, then follow the below instructions:

1. Go to the `cygwin` folder in your Files application.

> 2. Copy the path for `cygwin`.

Reviewer: Making it slightly more clear (?)
```suggestion
2. Copy the complete path to the `cygwin/bin` directory.
```

3. Right-click "This PC" on the left sidebar.
4. Select Properties > Advanced > Environment Variables.
5. Navigate towards the PATH variable in System variables, click "Edit".

> 6. Click "New", and add the `cygwin/bin` folder path as an environment variable.

Reviewer: (?)
```suggestion
6. Click "New" and paste the `cygwin/bin` directory path.
```

Author: slightly tweaked what you wrote
